packages/microservices/module/clients.module.ts

```diff
@@ -15,4 +20,71 @@ export class ClientsModule {
       exports: clients,
     };
   }
+
+  static registerAsync(options: ClientsModuleAsyncOptions): DynamicModule {
+    const clients: Provider[] = (options || []).reduce(
```

Comment:
```suggestion
    const clients: Provider[] = options.reduce(
```
I don't think we need this fallback?